build/ci/official.yml

```diff
@@ -10,7 +10,7 @@ resources:
 - repo: self
   clean: true
 queue:
-  name: VSEngSS-MicroBuild2019
+  name: VSEngSS-MicroBuild2019-1ES
```

Comment:
We recently switched to the `VSEngSS-MicroBuild2019` across the board. This was advised to us. I didn't see anything about this `-1ES` variant. There is more work to do than switching just this file if we need to do this across the board. Unfortunately, some of our pipelines (such as release pipelines) do not exist as YAML files in our repo. They are only in DevOps. Let me know if we are doing this for everything, and we can walk through it together.

Reply:
I can forward you the email. I was identified as the owner of the Dotnet-Project-System pipeline (🤷🏻‍♂️) which is why I got the notification.

Reply:
But the short answer is that `VSEngSS-MicroBuild2019` is being retired so any uses will need to be replaced.

Reply:
Got the email, thanks! Looks like it came from the same people that got us to do the original switch to scale sets. I understand. We'll update the file here and then do the other ones that need it too.